client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import handlers from './wpcom';
+
+/**
+ * WPCOM Middleware API
+ *
+ * Intercepts actions requesting data provided by the
+ * WordPress.com API and passes them off to the
+ * appropriate handler.
+ */
+export const middleware = store => next => action =>
+	// we won't use has( handlers, action.type )
+	// here becuase of performance implications
```

Comment:
thanks @aduth. I had read the same thing about a null prototype and wondered about this. also, this is one of the reason that I generally prefer the safer abstractions like `has` in non-performance-critical code. I'll create a PR to update